config/locale/de.json

```diff
@@ -226,7 +235,10 @@
     "Fire":"Feuer",
     "Flying":"Flug",
     "fog":"Nebel",
-    "form":"form",
+    "Forecast":"Vorhersage",
+    "form":"Form",
+    "formalola":"alolaform",
```

Comment:
I think this may break tracking

Reply:
ie formalola --> the regex expects form to be the parameter name, and then looks up the form type afterwards. 
ALthough maybe this means you can type alolaform and have it translated back into formalola which would work...

It makes my head hurt ;-)

Reply:
ok i removed it! now it should be fine